controlplane/src/core/workers/DeactivateOrganizationWorker.ts

```diff
@@ -0,0 +1,126 @@
+import { ConnectionOptions, Job, JobsOptions, Queue, Worker } from 'bullmq';
+import { PostgresJsDatabase } from 'drizzle-orm/postgres-js';
+import pino from 'pino';
+import * as schema from '../../db/schema.js';
+import { OrganizationRepository } from '../repositories/OrganizationRepository.js';
+import Keycloak from '../services/Keycloak.js';
+import { DeleteOrganizationQueue } from './DeleteOrganizationWorker.js';
+
+const QueueName = 'organization.deactivate';
+const WorkerName = 'DeactivateOrganizationWorker';
+
+export interface DeactivateOrganizationInput {
+  organizationId: string;
+  organizationSlug: string;
+  deactivationReason?: string;
+}
+
+export class DeactivateOrganizationQueue {
```

Comment:
I think it is time to ensure an interface for a worker implementation. Could you ensure that each worker implementation, implements an interface?

Reply:
Done